src/components/RentalFees/RentalFeeTs.ts

```diff
@@ -32,18 +32,17 @@ export class RentalFeeTs extends Vue {
         if (this.rentalEstimation) {
```

Comment:
when will this be unset?

Reply:
https://github.com/symbol/desktop-wallet/blob/7b5f35bf899d1f1e60d33fd4daab5ba3d684485e/src/store/Network.ts#L556-L563

hmm, I think most properly it won't be unset.
because when the network/node is down, the wallet won't work.
if the network is in working condition, it should always receive rental fees.

I will remove this `if` condition.